dotCMS/src/integration-test/java/com/dotcms/enterprise/cluster/ClusterFactoryTest.java

```diff
@@ -0,0 +1,21 @@
+package com.dotcms.enterprise.cluster;
+
+import com.dotcms.IntegrationTestBase;
+import com.dotcms.util.IntegrationTestInitService;
+import org.junit.Assert;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+public class ClusterFactoryTest extends IntegrationTestBase {
+
+    @BeforeClass
+    public static void prepare() throws Exception {
```

Comment:
Codacy found an issue: [A method/constructor should not explicitly throw java.lang.Exception](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5918493)